build.sh

```diff
@@ -288,7 +288,7 @@ function runWattsi {
 
 runWattsi $HTML_TEMP/source-whatwg-complete
 
-if [[ "$WATTSI_RESULT" -ne "0" ]]; then
+if [[ ! $QUIET = "true" || "$WATTSI_RESULT" -ne "0" ]]; then
```

Comment:
> "$QUIET" -ne "true" for consistency with the rest of the line?

Yeah I think I did in that way intentionally because I’m pretty sure bash `-ne` only works when comparing integer args. I’ll doublecheck.